grammar.js

```diff
@@ -2669,6 +2671,11 @@ module.exports = grammar({
       )
     ),
 
+    natural_join: $ => seq(
```

Comment:
you could also add `natural` as another choice in the `join` node which includes already `left`, `righter`... and so on instead of creating a new node `natural_join`

Reply:
Done.